src/test/java/seedu/address/model/appointment/AppointmentTest.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.model.appointment;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AppointmentTest {
+    private Appointment appointment1;
+    private Appointment appointment2;
+    private Appointment appointment3;
+    private LocalDateTime dateTime1;
+    private LocalDateTime dateTime2;
+
+    @BeforeEach
+    void setUp() {
+        dateTime1 = LocalDateTime.of(2024, 1, 1, 10, 0);
+        dateTime2 = LocalDateTime.of(2024, 1, 1, 11, 0);
+
+        appointment1 = new Appointment(new AppointmentType("Checkup"), dateTime1, 1,
+                new Sickness("Cold"), new Medicine("Aspirin"));
+        appointment2 = new Appointment(new AppointmentType("Checkup"), dateTime1, 1,
+                new Sickness("Cold"), new Medicine("Aspirin"));
+        appointment3 = new Appointment(new AppointmentType("Followup"), dateTime2, 2,
+                new Sickness("Flu"), new Medicine("Tamiflu"));
+        System.out.println(appointment1.toString());
+        System.out.println(appointment2.toString());
+        System.out.println(appointment3.toString());
+    }
+
+    @Test
+    public void testGetAppointmentType() {
```

Comment:
Not really a big issue, but do you think we can change the test method names to be consistent with what we already have? Currently it's in this format with the underscores.

<img width="401" alt="Screenshot 2024-10-16 at 10 13 30 AM" src="https://github.com/user-attachments/assets/aaf0fb54-407c-416f-8897-2168f6469d5d">